src/Theme.elm

```diff
@@ -185,7 +185,8 @@ pageHeadingStyle : Style
 pageHeadingStyle =
     batch
         [ fontSize (rem 1.8)
-        , margin2 (rem 2) zero
+        , outline none
```

Comment:
I guess this is ok?

Reply:
Yep. Fair, just worried that it was a 'hidden focus' - but I guess since it's -1 tab index won't confuse people as soon as they have pressed tab 1 time.